tests/Auth0.ManagementApi.IntegrationTests/DeviceCredentialsTests.cs

```diff
@@ -1,20 +1,31 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
 using System.Threading.Tasks;
+using System.Web;
 using Auth0.ManagementApi.Models;
+using Auth0.ManagementApi.Paging;
 using Auth0.Tests.Shared;
 using FluentAssertions;
+using Moq;
+using Moq.Protected;
+using Newtonsoft.Json;
 using Xunit;
 
 namespace Auth0.ManagementApi.IntegrationTests
 {
-    public class DeviceCredentialsTests : TestBase, IAsyncLifetime
+    public class DeviceCredentialsTests : TestBase
     {
-        private Client _client;
+        /*private Client _client;
         private Connection _connection;
         private User _user;
-        private const string Password = "4cX8awB3T%@Aw-R:=h@ae@k?";
+        private const string Password = "4cX8awB3T%@Aw-R:=h@ae@k?";*/
 
-        public async Task InitializeAsync()
+        /*public async Task InitializeAsync()
```

Comment:
I commented out all of the below code as it wasnt working.
Previously, it wasnt executed as the test in this file was marked to skip.
Now that I added a new test that is not being skipped, the `InitializeAsync` and `DisposeAsync` are getting executed aswell.

I tried fixing the integration tests, but no luck so far. I am having some issues with the endpoints, raised to the appropriate team internally.

For now, we can continue with a unit test, but I would love to circle back to the integration tests once this has been addressed internally.